src/llm/parser.py

```
@@ -0,0 +1,266 @@
+                        line = re.sub(r'^[-*•]\s*', '', line)  # Remove bullet points
+                        line = re.sub(r'^\d+\.\s*', '', line)  # Remove numbers
+                        conditions.append(line)
+                
+                return conditions[:5]  # Limit to 5 conditions
+            
+            return []
+            
+        except Exception:
+            return []
+    
+    def _validate_strategy(self, strategy: Dict) -> Optional[Dict]:
```

Pylint (3.3.7)

[refactor] Too many branches (13/12)

(R0912)